api/types/app.go

```diff
@@ -422,3 +423,86 @@ func (a Apps) Less(i, j int) bool { return a[i].GetName() < a[j].GetName() }
 
 // Swap swaps two apps.
 func (a Apps) Swap(i, j int) { a[i], a[j] = a[j], a[i] }
+
+const (
+	jwtClaimRewriteTypeNoneString           = "none"
+	jwtClaimRewriteTypeRolesString          = "roles"
+	jwtClaimRewriteTypeRolesAndTraitsString = "roles-and-traits"
+)
+
+func (r JWTClaimsRewriteType) encode() (string, error) {
```

Comment:
Is all of this extra code (basically all of the changes in this file) really worth it to be able to convert enum to/from string? Wouldn't it be simpler to just use string instead of a proto enum?

Reply:
I'm not sure if its worth it or not, its just the pattern I'd followed before, I'll change it to just a string